components/script/dom/node.rs

```diff
@@ -2370,7 +2390,11 @@ impl Node {
                 }
             }
         }
+
+        // Step 8. If suppress observers flag is unset, then queue a tree mutation record for parent
+        // with nodes, « », previousSibling, and child.
```

Comment:
Hrm. Is this actually happening? It might be worth a comment explaining why if not or a TODO if we are missing it.